scripts/count_runs.jl

```diff
@@ -0,0 +1,58 @@
+using DataFrames
+using BSON, FileIO, ValueHistories
+using Crayons
+using ArgParse
+s = ArgParseSettings()
+@add_arg_table s begin
+    "modelpath"
+    	required = true
+    	arg_type = String
+        help = "the path where model results are saved"
+    "--check"
+        help = "check validity of saved files"
+        action = :store_true
+end
+parsed_args = parse_args(ARGS, s)
+modelpath = parsed_args["modelpath"]
+check = parsed_args["check"]
+
+function list_files(datapath)
+	fs = readdir(datapath)
+	fs = filter(f->!(occursin("model_", f)), fs)
+	fs = filter(f->!(occursin("#", f)), fs)
+	return unique(fs)
+end
+function _check_validity(file)
+	data = load(file)
+	return (length(data[:val_labels]) == length(data[:val_scores])) && length(data[:tst_labels]) == length(data[:tst_scores])
```

Comment:
Checking `any(isnan.(___))` may be also useful.

Reply:
True, will add that.